src/ctapipe/tools/tests/test_calculate_pixel_stats.py

```diff
@@ -51,13 +58,13 @@ def test_calculate_pixel_stats_tool(tmp_path, dl1_image_file):
     # Check that the output file has been created
     assert monitoring_file.exists()
     # Check that the output file is not empty
-    assert (
-        read_table(
-            monitoring_file,
-            path=f"/dl1/monitoring/telescope/statistics/tel_{tel_id:03d}",
-        )["mean"]
-        is not None
-    )
+    mean_column = read_table(
+        monitoring_file,
+        path=f"/dl1/monitoring/telescope/statistics/tel_{tel_id:03d}",
+    )["mean"]
+    assert mean_column is not None
```

Comment:
this will never happen.

If `mean` doesn't exist, the line above raises an error that the column doesn't exist.